install.sh

```diff
@@ -83,6 +85,10 @@ main() {
         show $sudo eopkg update-repo -y
         show $sudo eopkg install -y brave
 
+    elif available rpm-ostree; then
```

Comment:
Should that be after `yum` maybe? Is it possible that a distro would have a working `yum` and also have `rpm-ostree`?